### ISSUE TITLE
[Bug] No longer able to define several font families

### ISSUE THREAD

**Issue by mlanglois** [OpenBoard-org/OpenBoard#1065]
### Describe the bug
I updated to 1.7.1 (from 1.6.3). I used to be able to define two font families in OpenBoardUser.config in this way:

```
[Board]
FontFamily="My main font, My secondary font"
```

### To Reproduce
1. Open OpenBoardUser.config
2. Go to [Board]
3. Add or change FontFamily= with two fonts installed in OpenBoard, separated by comma and enclosed in quotation marks
4. Launch OpenBoard and type text 

#### Expected behavior
OpenBoard used to display the text using the main font and, if the characters were not found (for instance using another alphabet that's not in the first font), it would use the secondary font.

#### Actual behavior
Now OpenBoard uses neither of these fonts, but the default font. Which means that OpenBoard does not recognize the setting at all. I tried without quotation marks, without comma, but it does not work.

### Context
  - reproducibility : `systematic` 
  - OpenBoard version :
      - bug happens with : `1.7.1`
      - bug doesn't happen with : `1.6.3`
      - didn't check : `1.6.4` `1.7.0`
  - OS : 
      - bug happens with : `macOS`
      - didn't check : `Windows` `Linux`  
  - Installation source : `OpenBoard download page`
  
### Additional context
- OS(es) detailed version (for the ones you reproduced the bug on) 
    - macOS : `macOS 14.6.1 (23G93) MacBook Pro 16-inch, 2019`
- number and resolution of monitors : `1 (1302x1024)`

**Comment by letsfindaway** [comment]
Theoretically, this should still work. Can you give a specific example of 
- font names, 
- the FontFamily setting 
- and a character string where this does not work as expected?

And be aware: as soon as you select a font using the font dialog in a text box, the `FontFamily` entry is overwritten and then contains the result of your selection. So this entry is more meant to remember the last selection and not for user configuration.

**Comment by mlanglois** [comment]
Sure! Here is the setting I've been using for years:
…
FontFamily="Gentium Book Plus, SBL Hebrew"
…

The second font includes Hebrew characters, which are not in the first
font. So, if I type something like:
"The Hebrew word מֶ֫לֶךְ means king"
 all characters are displayed with the main font (Gentium Book Plus) except
for the Hebrew word, which are displayed with the secondary font (SBL
Hebrew).

Now, with the latest version of OpenBoard, the default font is used instead.
As you mentioned, if I select a font using the dialog box, the setting is
overwritten. Fortunately I don't need to use it! ;)

**Comment by letsfindaway** [comment]
I just tried it on my system using

```
FontFamily="GE Typo Libre,Droid Sans Hebrew"
```

because the first is definitely a font without Hebrew and the second does not look like the default font. It works. With the default font:
![grafik](https://github.com/user-attachments/assets/a75568ce-efb7-4dcf-9fde-3e03886d2683)

With the font family as above:
![grafik](https://github.com/user-attachments/assets/16366902-270b-40ae-8321-d87cc42a9165)

You see the Hebrew characters look different in the second example.

So is it platform specific? I'm using openSUSE Linux here.

**Comment by letsfindaway** [comment]
I actually cannot tell what fonts are used. I have just seen that the two examples use different fonts. I also made a minimal HTML example:
```html
<html>
<head></head>
<body><p><span style="font-family:Adobe Courier, Droid Sans Hebrew; font-size:22pt">The Hebrew word מֶ֫לֶךְ means king</span></p></body>
</html>
```
and testes this in Firefox once with the `Droid Sans Hebrew` and once without. and got the results:

With Droid Sans Hebrew:
![grafik](https://github.com/user-attachments/assets/ce37a403-815c-4a63-8146-492e0fde6b5c)

Without Droid Sans Hebrew:
![grafik](https://github.com/user-attachments/assets/9b462e9a-641f-4072-98f7-c65ab06c1de2)

The fonts are different, that's all I can see. And they are exactly the same fonts I also get with OpenBoard.

**Comment by letsfindaway** [comment]
As I already said: for me it works also in OpenBoard. Two text boxes with and without the Droid Sans Hebrew as alternative font result in:
![grafik](https://github.com/user-attachments/assets/5eb50328-6f87-4bcc-bc6f-05beba899f98)

**Comment by mlanglois** [comment]
No, the screenshots you shared earlier from OpenBoard show that it did NOT
use Droid Sans Hebrew.

**Comment by letsfindaway** [comment]
> No, the screenshots you shared earlier from OpenBoard show that it did NOT use Droid Sans Hebrew.

Please double-check. Here a screenshot of my screenshots:
![grafik](https://github.com/user-attachments/assets/f3f23d07-5190-4c7f-beb3-af46b50fa2c5)

Exactly the same difference as in my latest tests.
